packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -176,11 +174,10 @@ export class RecipientPicker extends React.Component<RecipientProps> {
   }
 
   render() {
-    const { sections, listHeaderComponent, showContactSyncBanner } = this.props
+    const { sections, listHeaderComponent } = this.props
 
     return (
       <View style={style.body} testID={this.props.testID}>
-        {showContactSyncBanner && <ContactSyncBanner />}
```

Comment:
I have not found any other place we use `ContactSyncBanner`, if this is correct do we want to remove it?

Reply:
I was considering that but Nitya recommended we keep it, she suspects the new designs may still use it. I'll add a TODO on ContactSyncBanner to capture this